metabase/metabase
# Tweak boolean fields in basic action forms

Handles the case where a boolean form field has only title. Also adds stories to form components.

How to test:
- `yarn storybook` -> `FormToggle`
- Verify the label is centered when there is no description

Before
<img width="404" alt="Screenshot 2023-03-15 at 13 28 52" src="https://user-images.githubusercontent.com/8542534/225296370-0b3cb057-7635-48be-b9f6-6961b0db1d2e.png">

After
<img width="390" alt="Screenshot 2023-03-15 at 13 28 06" src="https://user-images.githubusercontent.com/8542534/225296403-64b426d2-12a0-46db-9e38-5cac1a6a7d7d.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29165)
